src/main/surfrdMod.F90

```diff
@@ -497,22 +503,26 @@ subroutine surfrd_special(begg, endg, ncid, ns)
 
     do nl = begg,endg
 
-       wt_lunit(nl,istdlak)     = pctlak(nl)/100._r8
+       wt_lunit(nl,istdlak) = pctlak(nl) / 100._r8
 
-       wt_lunit(nl,istwet)      = pctwet(nl)/100._r8
+       ! Until ctsm5.1 we would label ocean points as wetland in fsurdat
+       ! files. Starting with ctsm5.2 we label ocean points as ocean
+       ! (always 100%) and wetland points as wetland. Here we merge them
+       ! again to keep model behavior unchanged for now.
+       wt_lunit(nl,istwet) = (pctocn(nl) + pctwet(nl)) / 100._r8
```

Comment:
@swensosc:
As written now this exactly replicates what the code was doing before introducing pct_ocean.
What we want is
`wt_lunit(nl, istocn) = pctocn(nl) / 100._r8`
...and throughout the code we want to change istwet to istocn, not everywhere, but where Bill made changes for #1878 .

Reply:
@swensosc suggested starting at subr. apply_convert_ocean_to_land in surfrdUtilsMod.F90 [in Bill's PR merge](https://github.com/ESCOMP/CTSM/commit/075820c1902e19a586a1bdb8e31655123048360f#diff-2cf8bfd0e51dc881e54df728b909f59a91eafdefb381760c38f970f9ccbedd43)